airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -385,18 +385,25 @@ def attempt_task_runs(self):
                     )
                     self.pending_tasks.append(ecs_task)
                 else:
-                    self.send_message_to_task_logs(
-                        logging.ERROR,
-                        "ECS task %s has failed a maximum of %s times. Marking as failed. Reasons: %s",
-                        task_key,
-                        attempt_number,
-                        ", ".join(failure_reasons),
-                        ti=task_key,
+                    reasons_str = ", ".join(failure_reasons)
+                    self.log_task_event(
+                        record=Log(
+                            event="ecs executor queue error",
+                            task_instance=task_key,
+                            extra=(
+                                f"Task could not be queued after {attempt_number} attempts. "
+                                f"Marking as failed. Reasons: {reasons_str}"
+                            ),
+                        )
                     )
                     self.fail(task_key)
             elif not run_task_response["tasks"]:
-                self.send_message_to_task_logs(
-                    logging.ERROR, "ECS RunTask Response: %s", run_task_response, ti=task_key
+                self.log_task_event(
+                    record=Log(
+                        event="ecs runtime error",
```

Comment:
```suggestion
                        event="ecs execution error",
```

Reply:
does this mean the task ran and failed? or is it again some kind of submit failure?

Reply:
Yeah I think we can use `ecs task submit failure` here as well because it failed to submit the job on ECS